templates/default.yaml

```diff
@@ -10,14 +10,22 @@ STM32L496G_BG96_binary_step:
   0. Download the `Anjay-freertos-client-STM32L496G-BG96.bin` file.
 B_L462_CELL1_binary_step:
   0. Download the `Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin` file.
+nRF9160DK_board_name:
+  nRF9160 DK
+nRF7002DK_board_name:
+  nRF7002 DK
+nRF9160DK_flashing_tool_prerequisite:
+  Installed **nRF Connect** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-connect-for-desktop).
+nRF7002DK_flashing_tool_prerequisite:
+  Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download).
 zephyr_repository_step:
   0. Go to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
 nRF9160DK_binary_step:
   0. Download the `demo_nrf9160dk_merged.hex` file.
-nRF9160DK_flashing_nRF_docs_step:
-  0. To program the board, go through the whole process of **flashing nRF** [described here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160_gs.html).
+nRF9160DK_updating_dk_firmware_step:
+  0. You may need to update your DK firmware. To do this follow [Updating the DK firmware using Programmer](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf91/nrf9160.html#updating-the-dk-firmware-using-programmer) steps.
```

Comment:
I have replaced that with:

 0. You may need to update your DK firmware. To do this download new DK firmware from [nRF9160 DK Downloads page](https://www.nordicsemi.com/Products/Development-hardware/nRF9160-DK/Download#infotabs) and follow [Updating the modem firmware](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf91/nrf9160.html#updating-the-modem-firmware) steps.